changelog.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented here.
 ## [unreleased]
 
 Nothing yet.
+## [0.13.0] - Sept 26th, 2024
+- Fix bug where the `filter_annotations_on_load = true` option would not work as expected. 
+  - The `keypoint_slider_default_value` option was not being properly multiplied by 100 internally.
+- Move several config argument defaults into the configuration class instead of in the ULabel constructor.
+#### Breaking Changes
+- Deprecated the `default_toolbox_item_order` argument in the ULabel constructor. Use `toolbox_order` instead.
+- Deprecated `config_data` argument in the ULabel constructor. Instead, pass all configuration options as keyword arguments directly to the ULabel constructor.
+  - For now, the `config_data` argument will still work, but may be removed in a future release.
```

Comment:
idk if we'd ever remove support completely, but if we do we should prolly also drop support for the non-kwarg way of passing in constructor args as well

Reply:
Can we foresee any reason for logic grouping of certain config components? For example, `subtasks` and `on_submit` logic are generally going to be workflow-dependent, while keybinds might be user-dependent. I think we should either:
1. Have all configuration options be flat (which looks like what we're doing)
2. Provide reasonable logical groupings to streamline storage of certain data for those implementing ULabel in workflows
3. A hybrid approach, e.g. the constructor takes everything flat, but there's setters/getters for certain logical groups

(3) has some maintenance overhead, but will probably simplify the code on the user's end, assuming the logical groupings line up with how they would define them.

Reply:
yeah I moved to (1) because the `config_data` grouping we had seemed pretty arbitrary, and also this way we explicitly list all the config args in the `Configuration` class which provides stronger typing in the project,

currently we dont have any real support for post-init changing of keybinds and such, everything has always been set on init. The exception are the setters for `annotations` and the `image`. Having better setters for more of these configurable could be a nice feature down the line, and moves us towards (3).

Reply:
Opened #171 to track this. The more I think about it, the more use cases come up. 

Many tasks I've deployed have a need for a "power user mode", where the keybinds/display are totally different. ULabel itself doesn't need knowledge of this, but having specific setters to toggle modes like this would streamline that implementation.